.github/workflows/build-and-test.yml

```diff
@@ -44,6 +44,7 @@ jobs:
       matrix:
         build:
           - runs-on: runner
+        test_group_id: [1,2]
```

Comment:
Running in two groups will utilize 2 self-hosted runners. We can leave it like this (2 groups) or set to one group until we add more tests.

Note:
Until we implement .whl installer both runners will perform builds in parallel. 
Total workflow duration is same, but having whl installed will reduce runner utilization.